emscripten.py

```diff
@@ -114,8 +114,9 @@ def update_settings_glue(metadata, DEBUG):
   shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = sorted(set(all_funcs).difference(metadata['exports']))
 
   shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += metadata['globalImports']
-  # With the wasm backend the set of implemented functions is identical to the set of exports
-  shared.Settings.IMPLEMENTED_FUNCTIONS = [asmjs_mangle(x) for x in metadata['exports']]
+
+  all_exports = metadata['exports'] + list(metadata['namedGlobals'].keys())
+  shared.Settings.IMPLEMENTED_FUNCTIONS = [asmjs_mangle(x) for x in all_exports]
```

Comment:
All of this is for "implemented functions"... what about data symbols?

Reply:
namedGlobals are data symbols.  The name `IMPLEMENTED_FUNCTIONS` is misleading.. we should rename it `NATIVE_SYMBOLS` or something more meaningful.